src/test/scala/com/fluidcode/processing/silver/CreateCommentsTableSpec.scala

```diff
@@ -0,0 +1,83 @@
+package com.fluidcode.processing.silver
+
+import com.fluidcode.configuration.Configuration
+import com.fluidcode.models.bronze._
+import com.fluidcode.models.silver.SilverComments
+import com.fluidcode.processing.bronze.BronzeLayer
+import com.fluidcode.processing.silver.CreateCommentsTable._
+import org.apache.spark.sql.QueryTest
+import org.apache.spark.sql.delta.test.DeltaExtendedSparkSession
+import org.apache.spark.sql.test.SharedSparkSession
+
+class CreateCommentsTableSpec extends QueryTest
+  with SharedSparkSession
+  with DeltaExtendedSparkSession {
+
+  test("getCommentsData should select comments data from Bronze Layer" ) {
+
+    val sparkSession = spark
+    import sparkSession.implicits._
+    spark.conf.set(
+      "spark.sql.optimizer.excludedRules",
+      "org.apache.spark.sql.catalyst.optimizer.PushDownLeftSemiAntiJoin"
+    )
```

Comment:
what is this ?

Reply:
I used it to resolve the following error with Spark optimizer; 
> WARN: Max iterations (100) reached for batch Operator Optimization before Inferring Filters, please set 'spark.sql.optimizer.maxIterations' to a larger value.